discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -455,13 +455,12 @@ def get_node_ips_and_macs(self, node_name):
 
     @staticmethod
     def _get_domain_ips_and_macs(domain):
-        interfaces = dict(
-            # getting all DHCP leases IPs
-            **domain.interfaceAddresses(libvirt.VIR_DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE),
+        interfaces = dict()
+        
+        for addresses_source in (libvirt.VIR_DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE, libvirt.VIR_DOMAIN_INTERFACE_ADDRESSES_SRC_ARP):
```

Comment:
I will miss the comments I think, because usage of SRC_LEASE and SRC_ARP is not fully indicative about being relevant to dynamic and static networking

Reply:
Irrelevant to this PR.